### PR TITLE
add test for logs

### PR DESCRIPTION
**Pull request by chaitanyaphalak**
Added functional testing for logs ported over from SCK

### CODE REVIEW

File: .github/workflows/functional_test.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+name: Functional Test Helm Chart
+
+on: pull_request
+
+jobs:
+  e2e-test:
+    name: Functional Test Helm Chart - ${{ matrix.container_runtime }}
```
Comment (dmitryax) [inline .github/workflows/functional_test.yaml:7]: No need to repeat this in the job name. If we change to this:
```suggestion
    name: ${{ matrix.container_runtime }}
```
Github will show `Splunk Platform Functional Test / cri-o`

File: .github/workflows/functional_test.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          - "containerd"
+          - "cri-o"
+    env:
+      CI_SPLUNK_PORT: 8089
+      CI_SPLUNK_USERNAME: admin
+      CI_SPLUNK_HEC_TOKEN: a6b5e77f-d5f6-415a-bd43-930cecb12959
+      CI_SPLUNK_PASSWORD: helloworld
+      CI_INDEX_EVENTS: ci_events
+      CONTAINER_RUNTIME: ${{ matrix.container_runtime }}
+      KUBERNETES_VERSION: v1.21.2
```
Comment (dmitryax) [inline .github/workflows/functional_test.yaml:22]: Should add the last couple of major k8s versions into the matrix?

Not needed in this PR, just as follow-up suggestion. Up to you

File: test/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# SCK-OTEL Integration Tests Environment Setup
```
Comment (dmitryax) [inline test/README.md:1]:
```suggestion
# Splunk Platform Functional Test Environment Setup
```

File: test/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    export SPLUNK_HOST=$(kubectl get pod splunk --template={{.status.podIP}})
+    
+    # Use ci_scripts/sck_otel_values.yaml file to deploy sck otel connector
+    # Default image repository: quay.io/signalfx/splunk-otel-collector    
+    helm install sck-otel \
+    --set splunk_hec.index=ci_events \
+    --set splunk_hec.token=a6b5e77f-d5f6-415a-bd43-930cecb12959 \
+    --set splunk_hec.endpoint=https://$SPLUNK_HOST:8088/services/collector \
+    --set containers.containerRuntime=$CONTAINER_RUNTIME \
+    -f ci_scripts/sck_otel_values.yaml charts/sck-otel/
```
Comment (dmitryax) [inline test/README.md:57]: Please update instructions to use this helm chart

File: test/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
+# SCK-OTEL Integration Tests Environment Setup
+
+## Prerequsite
+* Python version must be > 3.x
+* Kubectl = v1.15.2
+* Minikube = v1.20.0
+* Helm = 3.3.x
+* libseccomp2 and cri-o (optional)
+## Setup local environment
+#### Start Minikube
```
Comment (dmitryax) [inline test/README.md:10]: Missing line breaks

File: ci_scripts/deploy_connector.sh [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+
+#Make sure to check and clean previously failed deployment
+echo "Checking if previous deployment exist..."
+if [ "`helm ls --short`" == "" ]; then
+   echo "Nothing to clean, ready for deployment"
+else
+   helm delete $(helm ls --short)
+fi
+echo "Deploying sck-otel-connect with latest changes"
```
Comment (dmitryax) [inline ci_scripts/deploy_connector.sh:12]:
```suggestion
echo "Deploying Splunk OTel Collector for Kubernetes"
```

File: .github/workflows/functional_test.yaml (hunk 2 above)
Comment (chaitanyaphalak) [reply .github/workflows/functional_test.yaml:22]: Yeah, in future.

**Review by dmitryax — APPROVED** [review]
LGTM

**Pull request merged**